docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -15,11 +15,15 @@ For information about writing queries to run with ``database analyze``, see
 
 Before starting an analysis you must:
 
-- :doc:`Set up the CodeQL CLI <getting-started-with-the-codeql-cli>` so that it can find the queries
-  and libraries included in the CodeQL repository.
+- :doc:`Set up the CodeQL CLI <getting-started-with-the-codeql-cli>` to run commands locally and
+  optionally check out the CodeQL repository if you want direct access to the CodeQL core queries
```

Comment:
Avoid highlighting the idea of checking out the repo here. If they are using a bundle or packs there is no need.

Reply:
Good point.